src/js/angular/ttyg/templates/chat-view.html

```diff
@@ -0,0 +1,28 @@
+<link href="css/ttyg/chat-view.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+<div class="chat-view">
+    <div class="messages" ng-class="{'hidden': !selectedChat}">
+        <div id="messages-scrollable">
+            <div class="messages-hint text-muted" ng-if="!selectedChat.messages.length">
+                {{ 'ttyg.hint' | translate }}
+            </div>
+            <div ng-repeat="question in selectedChat.messages" class="m-1"
+                 ng-class="{'mt-3': question.role === 'user'}">
+                <span ng-if="question.role === 'user'">User:  </span>
+                <span class="messages-item-content">{{ question.message }}</span>
+            </div>
+            <div onto-loader-new ng-show="loader" size="40"></div>
+        </div>
+    </div>
+    <div class="question">
+        <form>
+            <div class="mb-2 d-flex">
+                <input class="form-control" ng-model="chatQuestion.question" type="text">
+                <button class="btn btn-primary" ng-disabled="!chatQuestion.question" ng-click="ask()">
+                    <i class="fa fa-arrow-turn-left-up"></i>
```

Comment:
use the font awesome `regular` type icons!

Reply:
This will be done in a future request.

Reply:
Why?